cpp/ql/test/query-tests/Security/CWE/CWE-428/UnsafeCreateProcessCall.cpp

```diff
@@ -0,0 +1,503 @@
+// semmle-extractor-options: --microsoft
+#define NULL 0
+#define FALSE 0
+#define far
+#define LOGON_WITH_PROFILE 0x00000001
+
+typedef char CHAR;
```

Comment:
Would it be possible to remove most of these typedefs, and leave the various struct definitions with empty bodies (e.g. `struct _STARTUPINFOW { };`? This shouldn't affect the query, and all of these cut-and-paste definitions from the Windows headers kind of clutter up the real test cases.

Reply:
No problem. I have been trying to make the test function calls as close to the real ones, but I will change it.